src/AmbientSounds.Uwp/Controls/SettingsControl.xaml.cs

```diff
@@ -38,5 +45,45 @@ private void OnThemeSelectionChanged(object sender, SelectionChangedEventArgs e)
                 ViewModel.UpdateTheme(s);
             }
         }
+        
+        // Check if auto-startup is on and adjust its settings accordingly.
+        private async void AutoStartup_Toggled(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
+            StartupTask startupTask = await StartupTask.GetAsync("AmbieAutoStartup");
+            Debug.WriteLine(startupTask.State.ToString());
```

Comment:
Not a fan of slipping this in final code, imo.

Reply:
true, tysm